UpSkill/ClientSide/Infrastructure/AuthenticationService.cs

```diff
@@ -23,7 +23,7 @@ public async Task<RegistrationResponseDto> RegisterUser(UserRegistrationDto inpu
         {
             var content = JsonSerializer.Serialize(input);
             var bodyContent = new StringContent(content, Encoding.UTF8, "application/json");
-            var registrationResult = await client.PostAsync("accounts/registration", bodyContent);
+            var registrationResult = await client.PostAsync("accounts/register", bodyContent);
             var registrationContent = await registrationResult.Content.ReadAsStringAsync();
```

Comment:
I aim to learn with this question: have been wondering: 2 rows down we check whether the registrationResult returned was _unsuccessful_. If yes, we quit the work of the method. 

So, why do we read the registrationContent, before we know, if the registrationResult was successful?

Reply:
I see your point, I can't think of a reason why so I'll move it.

Reply:
** :) **